src/core/cmdline.ml

```diff
@@ -0,0 +1,79 @@
+(*
+ * Monad Module
+ * Durand-Marais © 2019
+ *)
+
+open Cmdliner
+open Monad
+
+let name = "eos"
+let version = "0.0"
+let licence = "MIT"
+let description = "a simple header manager for your projects.
+It allows to manage header in one single JSON file and automatize many features."
+let homepage = "https://github.com/maiste/eos"
+let bug_reports = "https://github.com/maiste/eos/issues"
+let dev_repo = "git://github.com/maiste/eos.git"
+let maintainer = [
+  "Etienne (Maiste) Marais <etienne@marais.green>";
+  "Xavier (Xavitator) Durand <xavier75013@gmail.com>"
+]
+let authors = [
+  "Etienne (Maiste) Marais <etienne@marais.green>";
+  "Xavier (Xavitator) Durand <xavier75013@gmail.com>"
+]
+
+let man =
+  let bug =
+    List.fold_left
+      (fun a b -> a @ [`Noblank; `P b])
+      [ `S Manpage.s_bugs;
+        `P ("Report bug with an issue at this url : " ^ bug_reports);
+        `P "Email them to maintainer(s) :"]
+      maintainer
+  in
+  let author = 
+    List.fold_left
+      (fun a b ->  a @ [`P b; `Noblank])
+      [`S Manpage.s_authors]
+      authors
+  in
+  let contribution =
+    [
+      `S "CONTRIBUTION";
+      `P ("$(b,Licence) : "^licence);
+      `P ("$(b,Homepage) : "^homepage);
+      `P ("$(b,Dev repository) : "^dev_repo)
+    ]
+  in
+  bug @ contribution @ author
+
+(* command corresponding to 'eos files' *)
+let cmd_files = 
+  let prompt_list l = (print_endline "list of tracked files :"; List.iter print_endline l) in
```

Comment:
Can you move it an externe function ? And use Printf.printf instead of print_endline ?